lib/parallel_tests/cli.rb

```diff
@@ -84,7 +85,24 @@ def run_tests_in_parallel(num_processes, options)
         report_time_taken(&run_tests_proc)
       end
 
-      abort final_fail_message if any_test_failed?(test_results)
+      if any_test_failed?(test_results)
+        warn final_fail_message
+
+        # Return the highest exit status. For BITs test there will are two known exit status/codes:
+        #
+        #  - 1 (generic test run failure)
+        #  - 69 (more specific, test run was unstable)
+        #
+        # By returning a more specific error status/code we can update other tooling to depend on that
+        # to report better status information on the test runs.
```

Comment:
```suggestion
        # return the highest exit status to allow sub-processes to send things other than 1
```

Reply:
Thanks for the tighter wording. Updated in 3a62cb0